api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -486,4 +486,10 @@ message CertificateValidationContext {
   // If this option is set to true, only the certificate at the end of the
   // certificate chain will be subject to validation by :ref:`CRL <envoy_v3_api_field_extensions.transport_sockets.tls.v3.CertificateValidationContext.crl>`.
   bool only_verify_leaf_cert_crl = 14;
+
+  // Config for the max number of intermediate certificates in chain that are parsed during verification
+  // This does not include the leaf certificate. If configured, and the certificate chain is longer than allowed, the certificates
+  // above the limit are ignored, and certificate validation will fail.
+  // https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_verify_depth.html
```

Comment:
Can you say something about what the default is? Even if you say it is system dependent, etc.?

Reply:
Will add it; yes it's system dependent